### PR TITLE
Improve MetalLB docs

### PR DESCRIPTION
**Pull request by Kristian-ZH**
Improve MetalLB docs and guides

### CODE REVIEW

File: asciidoc/quickstart/metallb.adoc [hunk 1]
```diff
@@ -0,0 +1,365 @@
+= MetalLB on K3s (using L2)
```
Comment (agracey) [inline asciidoc/quickstart/metallb.adoc:1]: This shouldn't be a quickstart. We need to add a new section for guides & how-to's, would you mind moving this file to asciidoc/guides/metallb-k3s.adoc?

File: asciidoc/components/metallb.adoc [hunk 2]
```diff
@@ -14,352 +14,25 @@ Via the https://metallb.universe.tf/[official docs]:
...
-* A K3s cluster where MetalLB is going to be deployed. Hint, you can use https://suse-edge.github.io/quickstart/k3s-on-slemicro[the K3s on SLE Micro guide].
-+
-[WARNING]
-====
-K3S comes with its own service load balancer named Klipper. You https://metallb.universe.tf/configuration/k3s/[need to disable it in order to run MetalLB]. To disable Klipper, K3s needs to be installed using the `--disable=servicelb` flag.
-====
-+
-* Helm
-* A couple of free IPs in our network range. In this case `192.168.122.10-192.168.122.12`
+A guide on installing MetalLB in front of the kube-api-server to achieve HA setups can be found in the link:../misc/metallb-kube-api.adoc[MetalLB in front of the Kubernetes API server] turotial.
```
Comment (agracey) [inline asciidoc/components/metallb.adoc:33]: The page referenced in this misc link should be moved to the guides section. I missed some clean up when porting over :(

File: asciidoc/components/metallb.adoc [hunk 3]
```diff
@@ -14,352 +14,25 @@ Via the https://metallb.universe.tf/[official docs]:
...
-In this quickstart, L2 mode will be used so it means we don't need any special network gear but just a couple of free IPs in our network range, ideally outside of the DHCP pool so they are not assigned.
+SUSE Edge utilizes MetalLB in two key ways:
 
-In this example, our DHCP pool is `192.168.122.100-192.168.122.200` (yes, 3 IPs, see <<traefik-and-metallb,Traefik and MetalLB>> for the reason of the extra IP) for a `192.168.122.0/24` network so anything outside this range is ok (besides the gateway and other hosts that can be already running!)
+* As a Load Balancer Solution: MetalLB serves as the Load Balancer solution for bare-metal machines.
+* For Achieving HA K3s/RKE2 Setup: MetalLB contributes to achieving High Availability setups with K3s/RKE2.
 
-=== Prerequisites
+== Best Practices
+Installation of MetalLB in L2 mode is detailed in link:../quickstart/metallb.adoc[the MetalLB quickstart].
```
Comment (agracey) [inline asciidoc/components/metallb.adoc:31]: Noted below that this page should move and be named metallb-k3s.adoc

**Review by agracey — APPROVED** [review]
Thank you for this! The content looks good, we just need to add a new section to house the referenced docs.

File: asciidoc/guides/metallb-k3s.adoc
Comment (agracey) [inline asciidoc/guides/metallb-k3s.adoc:1]: Metallb (not metal3) should be moved 😄

Comment (Kristian-ZH) [reply asciidoc/guides/metallb-k3s.adoc:1]: Opsss... should be fixed now 😄

**Review by agracey — APPROVED** [review]
LGTM

**Pull request merged**